features/waardetabel.feature

```diff
@@ -230,10 +253,12 @@ Functionaliteit: Waardetabel met code en omschrijving
       | land adres buitenland (13.10)     | Landen              | verblijfplaats               | verblijfadres.land      | 1234   | VerblijfplaatsBuitenland |
       | land vanwaar ingeschreven (14.10) | Landen              | immigratie                   | landVanwaarIngeschreven | 1234   |                          |
 
-  Rule: wanneer de waarde voor een plaats geen valide gemeentecode bevat wordt deze geleverd in de omschrijving
+  Rule: wanneer de waarde voor een plaats geen valide gemeentecode bevat wordt deze geleverd in de code en omschrijving
     - de waarde voor plaats is een gemeentecode wanneer het bestaat uit 4 cijfers
+    - als er in de geboorteplaats (03.20) een waarde staat die niet voorkomt in de landelijke tabel 34 Landen
```

Comment:
geldt dit alleen voor geboorteplaats of geldt het voor alle plaatsen (plaats overlijden, plaats in verblijfplaats)

Reply:
Aangepast. Dit geldt voor geboorteplaats, overlijdenplaats en plaats sluiting huwelijk/partnerschap.